### ISSUE TITLE
Missing the parameter for the specific Function.

### ISSUE THREAD

**Issue by mikesorensen5** [Azure/azure-cli#26473]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4246edfd-394e-4d61-360b-b95173159995
* Version Independent ID: e5365558-9b77-c72f-7b6d-f42e36a0a52c
* Content: [az functionapp keys](https://learn.microsoft.com/en-us/cli/azure/functionapp/keys?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/functionapp/keys.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/latest/docs-ref-autogen/functionapp/keys.yml)
* Service: **azure-functions**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by SaurabhSharma-MSFT** [comment]
@mikesorensen5 Can you please provide more details, which parameter is missing ?

**Comment by mikesorensen5** [comment]
Yes, the "Function" name parameter.  It has the FuntionApp, but not the Function.

Mike

[photo]
[photo]<https://www.wildenergyco.com/>
Mike Sorensen
President, Wild Energy

[icon] ***@***.***

[icon] www.wildenergyco.com<https://www.wildenergyco.com/>

[icon] (312) 685-8811

[icon] (833) 563-6300

[icon] 935 N Beneva Road, Suite 609-18 Sarasota, FL 34232<https://maps.google.com/?q=935%20N%20Beneva%20Road,%20Suite%20609-18%20Sarasota,%20FL%2034232>

[facebook]<https://www.facebook.com/WildEnergyCo/>      [linkedin] <https://www.linkedin.com/company/wildenergyco/>
[Custom Button App]<https://wildenergyco.zohobookings.com/#/customer/mikes>

[https://tracy.srv.wisestamp.com/px/wsid/D0nznpp0.png]
________________________________

**Comment by kamperiadis** [comment]
Hi @mikesorensen5! I just want to confirm - so you want to run `az functionapp keys` for a function within a function app? If that is the case, we have the [az functionapp function keys](https://learn.microsoft.com/en-us/cli/azure/functionapp/function/keys?view=azure-cli-latest) commands for that. Can you please confirm if this is what you were looking for? Thank you.
